holoviews/plotting/plotly/callbacks.py

```diff
@@ -0,0 +1,215 @@
+from weakref import WeakValueDictionary
+
+from param.parameterized import add_metaclass
+
+from ...streams import (
+    Stream, Selection1D, RangeXY, RangeX, RangeY, BoundsXY, BoundsX, BoundsY
+)
+
+from .util import _trace_to_subplot
+
+
+class MetaPlotlyCallback(type):
```

Comment:
As a quick comment, I would prefer the name `PlotlyCallbackMetaClass` because the name alone just suggests it is meta, not specifically a metaclass.

Reply:
No objection.